tiffany-tiffanyteaze/week-3/giffaw/scripts/app.js

```diff
@@ -0,0 +1,31 @@
+let giphyUrl = "http://api.giphy.com/v1/gifs/search?q=";
+let apiKey = 'mIyOSO9AfIEquW3Daf5YzSdJ5PmOvG6R';
+
+$( document ).ready(function() {
+  $('form').on('submit', function (e) {
+    e.preventDefault();
+    console.log("You clicked submit.");
```

Comment:
Great sanity check. I do this myself just to ensure event handling is occurring. Just make sure to remove all console logs once complete with a project. :octocat: